index.js

```diff
@@ -78,15 +104,48 @@ function getServices(offset) {
           bdk.log.warn('service missing name', service);
         }
       } else {
-        bdk.log.warn('broken service', service);
-        bdk.log.warn('full list of services', services);
+        bdk.log.warn('broken service');
       }
     });
 
     return serviceMap;
   });
 }
 
+/**
+ * Get all incident timestamps
+ *
+ * @param {Array} pdData - list of pagerduty data
+ * @returns {Array} - All the ttes
+ */
+export function getIncidents(pdData) {
+  const tteList = [];
+  return new Promise((resolve) => {
+    // eslint-disable-next-line consistent-return
+    Promise.all(pdData.map((obj) => {
+      if (obj.incident) {
+        return pdIncidentDetail(obj.incident.id).then((result) => {
```

Comment:
The block after .then can be tested

Reply:
You can abstract this block and given an expected JSON you will expect a list of TTE.